changelog/unreleased/issue-4103

```diff
@@ -0,0 +1,12 @@
+Bugfix: fix restic dump of tar file with 32-bit binary
+
+In restic up to 0.14.0, the restic dump from a 32-bit binary of a
```

Comment:
I'd reword it as follows:
```
When using a 32-bit build of restic, the `restic dump` command could in some
cases create tar files containing negative uid and gid. These files cannot be
read by gnu tar. This corner case especially applies to backups from stdin on Windows.
We've changed the dump command to create valid tar files in this case.
```

Reply:
I agree. Please check the latest commit @MichaelEischer.